src/main/kotlin/com/jetbrains/snakecharm/lang/psi/SmkFileReference.kt

```diff
@@ -11,21 +11,24 @@ import com.jetbrains.python.psi.PyStringLiteralExpression
 class SmkFileReference(
         element: PsiElement, textRange: TextRange
 ) : PsiReferenceBase<PsiElement>(element, textRange) {
+    private val key: String = element.text.substring(textRange.startOffset, textRange.endOffset)
+
     override fun resolve(): PsiElement? {
-        if (element !is PyStringLiteralExpression) {
-            return null
-        }
+        val stringLiteral = (element as? SMKWorkflowParameterListStatement)?.
```

Comment:
If reference supports only  `SMKWorkflowParameterListStatement` let's change it's constructor to accept only `SMKWorkflowParameterListStatement` psi elements instead of `PsiElement`